package.json

```diff
@@ -4,7 +4,13 @@
   "description": "Action to handle automatic PR labeling",
   "main": "dist/index.js",
   "scripts": {
-    "build": "ncc build src/index.ts --out dist"
+    "test": "jest",
+    "build": "ncc build src/index.ts --out dist",
+    "prepare": "husky install"
+  },
+  "engines": {
+    "node": "12.x",
+    "npm": "6.x"
```

Comment:
The built-in npm with `node12` is v6, so `package-lock.json` version is v1. I want to avoid that some contributor upgrade the package-lock to v2 by accident.

Reply:
Looking at the docs, this doesn't actually prevent anything, just shows a warning, and they can still commit. Honestly, this field is pointless, and you're better off making it part of the CI to ensure that no one is checking in a v2 lockfile.